pkgs/applications/networking/protonvpn-gui/default.nix

```diff
@@ -92,12 +94,14 @@ buildPythonApplication rec {
   # Gets a segmentation fault after the widgets test
   doCheck = false;
 
-  meta = with lib; {
+  passthru.updateScript = nix-update-script { };
+
```

Comment:
```suggestion
```
We don't need to set such a simple update-script because the python builder has its own update-script.
Please apply to all other changes.
https://github.com/NixOS/nixpkgs/blob/00f536f650269d876d5a7c8dcc8bb567ee4df233/pkgs/development/interpreters/python/mk-python-derivation.nix#L218-L224

Reply:
But why aren't there any PRs for the existing updates triggered?

Reply:
This appears to be only working for python-libs that are published via some kind of dependency-hub not only via github - can you confirm?

For me I need this that the `maintainers/scripts/update.nix` recognizes new releases on GitHub for the package e.g. `nix-shell maintainers/scripts/update.nix --argstr package python3Packages.proton-vpn-api-core`

Reply:
Sorry, I noticed that python-update-libraries checks github releases but not tags.
That's why it did not update this package.

https://github.com/NixOS/nixpkgs/blob/3c799ef618f8a0ea383898b7e8877fcec4783988/pkgs/development/interpreters/python/update-python-libraries/update-python-libraries.py#L265-L267

I will fix this, but it's not a blocker.

Reply:
Thanks for letting me know and taking care of this - will remove the update-scripts then :)